class-tgm-plugin-activation.php

```diff
@@ -371,6 +380,9 @@ public function init() {
 					'The following recommended plugins are currently inactive: %1$s.',
 					'tgmpa'
 				),
+				'notice_force_activation'        => _n_noop(
+					'The following plugin has been automatically activated because it is required by the current theme: %s', 'The following plugins have been automatically activated because they are required by the current theme: %s'
```

Comment:
This string will also need to be added to the `example.php` file for potential overruling.